tools/posix/generate_breakpad_symbols.py

```diff
@@ -198,20 +188,24 @@ def _Worker():
                 elif sys.platform == 'linux2':
                     binary = GetSymbolPath(options, binary)
 
-                syms = GetCommandOutput([GetDumpSymsBinary(options.build_dir), '-r', '-c', binary])
-                module_line = re.match("MODULE [^ ]+ [^ ]+ ([0-9A-F]+) (.*)\n", syms)
-                output_path = os.path.join(options.symbols_dir, module_line.group(2),
+                dump_syms_bin = options.dump_syms_bin
+                if not os.access(dump_syms_bin, os.X_OK):
+                    raise Exception(f'Cannot find dump_syms: {dump_syms_bin}')
+
+                syms = GetCommandOutput([dump_syms_bin, '-c', binary])
```

Comment:
`-r` is removed (` Do not handle inter-compilation unit references`).
With the flag there are a lot of errors and the resulting symbols seems to be nearly empty.